sql_script/initialize_db.sql

```diff
@@ -0,0 +1,455 @@
+CREATE TABLE personnel ( 
+    id INTEGER, 
+    birth_date DATE, 
+    first_name TEXT, 
+    last_name TEXT, 
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE line_worker ( 
+    id INTEGER, 
+    responsibilities TEXT, 
+    PRIMARY KEY (id),
+    FOREIGN KEY (id)
+        REFERENCES personnel (id)
+        ON DELETE CASCADE
+        ON UPDATE CASCADE
+);
+
+CREATE TABLE inventory_manager ( 
+    id INTEGER,
+    PRIMARY KEY (id),
+    FOREIGN KEY (id)
+        REFERENCES personnel (id)
+        ON DELETE CASCADE
+        ON UPDATE CASCADE
+);
+CREATE TABLE personnel_manager (
+    id INTEGER, 
+    number_of_employees_managed INTEGER, 
+    PRIMARY KEY (id),
+    FOREIGN KEY (id)
+        REFERENCES Personnel (id)
+        ON DELETE CASCADE
+        ON UPDATE CASCADE
+);
+
+CREATE TABLE dimensions (
+    height DOUBLE PRECISION,
+    width DOUBLE PRECISION,
+    length DOUBLE PRECISION,
+    volume DOUBLE PRECISION,
+    PRIMARY KEY (height, width, length)
+);
+
+CREATE TABLE region( 
+    postal_code CHAR(6), 
+    municipality TEXT, 
+    province_or_territory TEXT, 
+    PRIMARY KEY (postal_code)
+);
+
+CREATE TABLE difference_cache( 
+    larger DOUBLE PRECISION,
+    smaller DOUBLE PRECISION,
+    difference INTEGER NOT NULL, 
```

Comment:
you'll want to make `difference` a `double precision` as well :)
see: https://github.com/nuxeric/cpsc304project/pull/3#discussion_r523468059